operator/README.md

```diff
@@ -25,10 +25,10 @@ v2.3 | v4.10+ | 6.5+ | 4.4.9+
 
 The operator will install all the necessary components which Forklift needs to operate. The projects and a description of each are detailed below:
 
-* [Forklift UI](https://github.com/konveyor/forklift-ui), The Forklift UI is based on [Patternfly 4](https://www.patternfly.org/v4).
-* [Forklift Controller](https://github.com/konveyor/forklift-controller), The Forklift Controller orchestrates the migration.
-* [Forklift Validation](https://github.com/konveyor/forklift-validation), The Forklift Validation service checks the VMs for possible issues before migration. This service is based on [Open Policy Agent](https://www.openpolicyagent.org).
-* [Forklift Must Gather](https://github.com/konveyor/forklift-must-gather), Support tool for gathering information about the Forklift environment.
+* [Forklift UI](https://github.com/kubev2v/forklift-ui), The Forklift UI is based on [Patternfly 4](https://www.patternfly.org/v4).
+* [Forklift Controller](https://github.com/kubev2v/forklift-controller), The Forklift Controller orchestrates the migration.
+* [Forklift Validation](https://github.com/kubev2v/forklift-validation), The Forklift Validation service checks the VMs for possible issues before migration. This service is based on [Open Policy Agent](https://www.openpolicyagent.org).
```

Comment:
Controller (above) and validation should now point to `kubev2v/forklift`. Or in the respective sub directories of it.

Reply:
right, changed